app/styles/_core.less

```diff
@@ -1286,3 +1286,13 @@ copy-to-clipboard .input-group.limit-width {
     width: 100%;
   }
 }
+
+// Within truncation directive controls spacing between content and toggle link
+.truncation-block {
+  margin-right: 10px;
+}
+
+.truncated-content-toggle-link {
```

Comment:
I'd just call this `.truncation-collapse-link` since it is no longer for the see all link.